DEV.md

```diff
@@ -6,6 +6,7 @@ This guide targets Developers of this SDK. It explains how to build the project
 ---
 
 * [How to Build the Sample App](#build-sample-app)
+  * [How to configure sample app at build time](#configure-sample-app) 
```

Comment:
I guess this configure-sample-app ID is not added anywhere in the doc, can you please check?